substrate-archive-gateway/metadata/databases/HydraIndexer/tables/public_substrate_event.yaml

```diff
@@ -13,6 +13,10 @@ configuration:
     extrinsic_index: extrinsicIndex
     extrinsic_name: extrinsicName
     index_in_block: indexInBlock
+    evm_log_address: evmLogAddress
```

Comment:
Are we going to have these columns for all archives or only evm supported?

Reply:
Definitely only for evm.

Reply:
Let's keep archive changes out of this repo for the moment.